fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -594,6 +594,18 @@ private static int countLinkHeader(final HttpResponse response, final String uri
         }).count();
     }
 
+    protected static String getOriginalResourceUri(final CloseableHttpResponse response) {
+        final List<String> linkList = getLinkHeaders(response).stream().filter(x -> {
+            final Link theLink = Link.valueOf(x);
+            return theLink.getRel().equals("original");
+        }).map(x -> Link.valueOf(x).getUri().toString()).collect(toList());
+
+        assertTrue("No original resource Link in headers", linkList.size() != 0);
```

Comment:
I think it'd be okay to just use the second assert (equals 1), and also switch to assertEquals. Or skip the assertions and have the stream block return findAny without validating the number of headers since this is a test helper, up to you.